Pod/Classes/VOKCoreDataManager.m

```diff
@@ -161,8 +145,7 @@ - (void)initPersistentStoreCoordinator
                               NSInferMappingModelAutomaticallyOption: @YES,
                               };
     
-    NSError *error;
-    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:[self managedObjectModel]];
+    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:self.managedObjectModel];
```

Comment:
I think they already function as lazy getters: they are only called from the get methods for `persistentStoreCoordinator` and `managedObjectModel`.  Can you explain more what you would change?